2020/02/28/consolidate-multiple-sftp-accounts-on-one-master-account.md

```diff
@@ -0,0 +1,153 @@
+---
+author: "Selvakumar Arumugam"
+title: "Consolidate Multiple SFTP Accounts on One Master Account"
+tags: ssh, sftp, shell
+---
+
+
+Recently, a data intensive client implemented a workflow process to generate various reports and insights from a list of facilities. Because a significant portion of these files contain sensitive data, they would need to strictly adhere to HIPAA compliance. Optimally, facilities should be able to transfer files securely and exclusively to our server. Arguably, the best method of achieving this is to create individual SSH File Transfer Protocol (SFTP) accounts for each source.
+
+
+### SFTP Account
+
+Private SFTP accounts were established for each facility and the data was received at a designated path. At these individual points of contact, a third party application picks up the data and processes further into the pipeline. The following demonstrates how SFTP accounts are developed and configured:
+
+
+* Create a user group for SFTP accounts
+
+```bash
+$ addgroup sftpusers
+```
+
+* Configure the following settings in sshd_config (this enables an SFTP account and defaults the location as the home path)
+
+```bash
+$ vi /etc/ssh/sshd_config
+
+Subsystem       sftp    internal-sftp
```

Comment:
It almost looks like this is the entirety of the sshd_config file. Maybe something should be there, even just a "<snip>", to imply what's above has been cut out for brevity and this is just the portion that's added/changed.

Reply:
Done